packages/components/src/form-token-field/test/index.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint jest/expect-expect: ["warn", { "assertFunctionNames": ["expect", "expectTokensToBeInTheDocument", "expectTokensNotToBeInTheDocument", "expectVisibleSuggestionsToBe", "expectEscapedProperly"] }] */
```

Comment:
We're intentionally extending the list of assert functions since we're using some custom ones in this suite.